iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/StaticVarCompensatorXml.java

```diff
@@ -85,15 +103,12 @@ protected StaticVarCompensator readRootElementAttributes(StaticVarCompensatorAdd
                 .setBmax(bMax)
                 .setVoltageSetpoint(voltageSetpoint)
                 .setReactivePowerSetpoint(reactivePowerSetpoint)
-                .setRegulationMode(regulationMode);
+                .setRegulationMode(StaticVarCompensator.RegulationMode.OFF);
         readNodeOrBus(adder, context);
-        StaticVarCompensator svc = adder.add();
-        readPQ(null, svc.getTerminal(), context.getReader());
-        return svc;
+        return regulationMode;
     }
 
-    @Override
-    protected void readSubElements(StaticVarCompensator svc, NetworkXmlReaderContext context) throws XMLStreamException {
+    private void readSubElementsStaticVarCompensator(StaticVarCompensator svc, NetworkXmlReaderContext context) throws XMLStreamException {
```

Comment:
Same remark

Reply:
Finally, the standard structure of reading has been preserved.